Cron Step 4.5: rebase-pr.sh returned REBASE_NOOP — branch is current with develop. Failure is real.

```
=== job 74081867173 ===
2026-05-03T01:21:08.2658471Z FAIL	github.com/cfgis/cfgms/features/siem [build failed]
=== job 74081867184 ===
2026-05-03T01:22:58.2431830Z FAIL	github.com/cfgis/cfgms/features/siem [build failed]
=== job 74081867185 ===
2026-05-03T01:28:59.6026492Z panic: test timed out after 5m0s
2026-05-03T01:29:00.5162018Z FAIL	github.com/cfgis/cfgms/features/controller/api	300.675s
2026-05-03T01:29:36.6483176Z FAIL	github.com/cfgis/cfgms/features/siem [build failed]
2026-05-03T01:31:00.9815506Z --- FAIL: TestSessionDataHandling (0.03s)
2026-05-03T01:31:00.9839587Z FAIL
```

Story labelled `pipeline:fix`; next cron cycle will dispatch the fix agent.

--- a/features/siem/siem_engine.go
+++ b/features/siem/siem_engine.go
@@ -421,7 +421,6 @@ func (se *SIEMEngine) GetMetrics(ctx context.Context) (*SIEMMetrics, error) {
 	// Read atomically-updated fields outside the lock using atomic.Load*.
 	snap.TotalEntriesProcessed = atomic.LoadInt64(&se.metrics.TotalEntriesProcessed)
 	snap.DroppedEntries = atomic.LoadInt64(&se.metrics.DroppedEntries)
-	snap.ActiveWorkers = atomic.LoadInt32(&se.metrics.ActiveWorkers)
 	snap.ProcessingErrors = atomic.LoadInt64(&se.metrics.ProcessingErrors)
 	snap.PatternsMatched = atomic.LoadInt64(&se.metrics.PatternsMatched)
 	snap.SecurityEventsGenerated = atomic.LoadInt64(&se.metrics.SecurityEventsGenerated)

